src/cf-forms/usage.md

```diff
@@ -354,10 +367,277 @@ Provides sizeable margins between groups of form elements.
 <div class="btn-inside-input">
     <input class="input__super"
            type="text"
-           value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable." title="Test input">
+           value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable."
+           title="Test input">
     <button class="btn btn__super btn__link btn__secondary">
         Clear
         <span class="cf-icon cf-icon-delete"></span>
     </button>
 </div>
 ```
+
+### Checkboxes with large target areas
```

Comment:
I'd remove up to line 534 since that's refactored in #384. Otherwise we can make a note to fix in another PR.